examples/bootstrap/versions.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">=0.12.29, <0.15"
+  required_version = ">=0.12.29, <0.16"
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
```

Comment:
Why aren't we pinning the `azurerm` provider to version `2.58` here? From what I've understood, every version older than 2.46 will be pinned that way. Please ignore the following comments if I didn't understood your intention correctly.